import-export-cli/cmd/keys.go

```diff
@@ -81,12 +83,19 @@ func getKeys() {
 	tiers, err := getAvailableAPITiers(accessToken)
 
 	if tiers != nil && err == nil {
-		utils.Logln(utils.LogPrefixInfo + "Retrieved available subscription tiers of the API: ", tiers)
-		//Needs an available subscription tier when creating application
-		throttlingTier = tiers[0]
+		utils.Logln(utils.LogPrefixInfo+"Retrieved available subscription tiers of the API: ", tiers)
+		// Needs an available subscription tier when subscribing to the particular API using the application
+		subscriptionThrottlingTier = tiers[0]
 	} else {
 		utils.HandleErrorAndExit("Internal error occurred", err)
 	}
+	// Retrieving application throttling policy
+	applicationThrottlingPolicy, err := getApplicationThrottlingPolicy(accessToken)
+	// If the application throttling policy call fails, exit with the error
+	if err != nil {
+		utils.HandleErrorAndExit("Internal error occurred", err)
+	}
+	utils.Logln(utils.LogPrefixInfo+"Retrieved application throttling policy successfully: ", applicationThrottlingPolicy)
```

Comment:
Shall we add a space between LogPrefixInfo+"Retrieved application throttling policy successfully: "?